### PR TITLE
[IMP] mail: chatter: remove insert_message method, instead of push ea...

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
...ch message to DOM using insert_message, create render_thread that will render each of it's message
- Task: https://www.odoo.com/web#id=8546&view_type=form&model=project.task&action=333&active_id=131
- Pad: https://pad.odoo.com/p/r.AdsUG2IZU0s39ry8

### CODE REVIEW

File: addons/mail/static/src/js/mail.js [hunk 1]
```diff
@@ -1489,70 +1447,64 @@ openerp.mail = function (session) {
...
-                message.insertAfter(dom_insert_after);
+                _(messages).invoke('insertAfter', dom_insert_after)
             } else if (prepend) {
-                message.prependTo(self.$el);
+                _(messages).invoke('prependTo', self.$el)
             } else {
-                message.appendTo(self.$el);
+                _(messages).invoke('appendTo', self.$el)
             }
-            message.$el.hide().fadeIn(500);
```
Comment (Gorash) [inline addons/mail/static/src/js/mail.js]: Should we remove the effect?

Comment (aja-odoo) [inline addons/mail/static/src/js/mail.js]: yes , why we will wait 500 millisecond to display thread, staring as soon as messages arrived

**Comment by rim-odoo** [comment]
@hmo-odoo Closed?

**Comment by hmo-odoo** [comment]
@rim-odoo : Ajay is working on another branch (https://github.com/odoo-dev/odoo/tree/8.0-optimize-chatter-aja). so I closed this branch

**Comment by rim-odoo** [comment]
@hmo-odoo :+1: (I didn't noticed the branch change)

**Pull request closed without merging**